### DISCUSSION TITLE
Device Tab - Connection to printer (Bambu X1C) is flapping in 1.9.0

### DISCUSSION THREAD

**Discussion by WebbuffUSA** [OrcaSlicer/OrcaSlicer#3604] [General]
After upgrading to 1.9.0 I get the following error when under the Device tab of Orca Slicer 1.9.0:"Failed to connect to printer"

This connection to the device is "flapping" from connected to not connected (with the error) so it is not a constant state.

I am able to slice and send prints to the printer and the printer (X1C) prints the file fine.

I have no issues with the Handy app for this printer, nor is my wife's laptop having the same issue.

When I re-install 1.8.1, the issue does not occur. Here are the steps I have taken to attempt to resolve this issue.
- Rebooted the printer multiple times.
- Uninstalled and re-installed 1.9.0.
- Deleted the plugins folder in the appdata folder to force the network plug-in to reinstall.
- Removed (forgot) wifi connection to PC and re-added.
- Logging out and back into the printer from Orca Slicer
- Uninstalled 1.9.0 completely and deleted any remaining folders and re-installed.
- Installed 1.8.1 and re-tested (works fine).
- Checked for other reports of similar issues on Reddit/Google.
- Ensured firewall had the proper exception for this application (it did).
- Started application and waited 60 seconds before moving to the Device tab.

Not sure what else to try or if this is just a bug and I need to wait for next patch.

**Comment by troygiese** [comment]
1.9.0 was working flawlessly for me until this morning (Feb 9, 2024).  I can slice and send the job to the cloud and successfully print, but it cannot connect to the printer on the device tab.

It makes some sort of connection before the error message - the printer physically responds (sounds like a stepper chirp).

**Comment by LightsOnArborWay** [comment]
having same problem with 1.90  keeps going from connected to not connected.

**Comment by MK-ultr-a** [comment]
Just started having this issue as well after a clean windows install, Latest version of BS is fine however and same with 1.8.1

**Comment by WebbuffUSA** [comment]
Weirdly enough, it stopped occurring after not using Orca Slicer for a couple weeks. I then update to 1.9.1 and was fine for a day, but now the issue has returned. I re-tried disconnecting and reconnecting the printer in Orca and also deleting the Plugins folder and re-installing the Network Plug-in. No change.
